### PR TITLE
Update:  theming and intro screen

### PR DESCRIPTION
**Pull request by dpaez**
This PR adds some branding theming tweaks and a new intro "splash" screen that redirects to the app.

### CODE REVIEW

File: packages/dashboard/public/index.html [hunk 1]
```diff
@@ -25,6 +25,11 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Permanent Seeder Dashboard</title>
+    <script>
+      window.loadPromise = new Promise(function(resolve) {
+        window.addEventListener('load', resolve);
+      });
+    </script>
```
Comment (estebanprimost) [inline packages/dashboard/public/index.html]: I think we can remove this from here. You can move it to `timeline.js`
```suggestion
```

File: packages/dashboard/src/timeline.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+export const play = (pathname, node, appears) => {
+  if (!node) return
+  const delay = appears ? 0 : 0.5
+  let timeline
+
+  if (pathname === '/') { timeline = getHomeTimeline(node, delay) } else { timeline = getDefaultTimeline(node, delay) }
+
+  window
+    .loadPromise
+    .then(() => window.requestAnimationFrame(() => {
```
Comment (estebanprimost) [inline packages/dashboard/src/timeline.js]: From `public/index.html`:
```suggestion
  const loadPromise = new Promise(function (resolve) {
    window.addEventListener('load', resolve)
  })

  loadPromise.then(() => window.requestAnimationFrame(() => {
```

File: packages/dashboard/src/containers/App.js [hunk 3]
```diff
@@ -1,27 +1,105 @@
...
+import CssBaseline from '@material-ui/core/CssBaseline'
 
 import Layout from '../components/Layout'
 
 import { AppStateProvider } from '../context/app-state'
 
 import Dashboard from './Dashboard'
+import SplashScreen from '../components/SplashScreen'
+import NoMatch from '../components/NoMatch'
+import { play, exit } from '../timeline'
```
Comment (estebanprimost) [inline packages/dashboard/src/containers/App.js]: All good, just import sort.
```suggestion
import { play, exit } from '../timeline'

import { AppStateProvider } from '../context/app-state'

import Layout from '../components/Layout'
import NoMatch from '../components/NoMatch'
import SplashScreen from '../components/SplashScreen'

import Dashboard from './Dashboard'
```

**Review by estebanprimost — APPROVED** [review]
Good feature!

**Comment by dpaez** [comment]
PR Updated

**Pull request merged**